src/app/error-page/error-page.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-error-page',
+  templateUrl: './error-page.component.html',
+  styleUrls: ['./error-page.component.scss']
+})
+export class ErrorPageComponent implements OnInit {
+
+  constructor(private router: Router) {}
+
+  ngOnInit(): void {
+    setTimeout(()=>{
```

Comment:
better to include a link for a user so he can navigate to the courses page himself =)
Just a suggestion

Reply:
@SteveRusin Yes, I agree with you. It would be better. I changed it to link for user =)